Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -18,8 +18,10 @@ - (void)getRequestAsync:(NSDictionary *)params url:(NSString *)url andTag:(NSStr
     [self getRequestAsync:params url:url andTag:requestTag log:YES];
 }
 
+// this is actually a synchronous call, since its callers are NOT on the main queue
```

Comment:
If these two calls are actually synchronous, renaming them would probably be best. They're not publicly exposed anyway, right?